Source/SimCacheModule/Source/Core/ProgressionData.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2024 Steven Frost and Orion Lyau
+
+#pragma once
+
+#include "Core/CacheDefinitionCollection.h"
+
+#include <Utils/Serialisation/Serialisable.h>
+#include <Utils/Time/DateTime.h>
+
+#include <vector>
+
+// -----------------------------------------------------------------------------
+
+struct AcquiredCacheMetadata
+	: public Utils::Serialisation::ISerialisable
+{
+public:
+
+	AcquiredCacheMetadata();
+	AcquiredCacheMetadata( const CacheId& Id, const Utils::DateTime& TimeAcquired );
+
+public: // ISerialisable
+
+	virtual bool Serialise( Utils::Serialisation::Writer& Writer ) const override;
+	virtual bool Deserialise( Utils::Serialisation::Reader& Reader ) override;
+
+public:
+
+	CacheId			Id;
+	Utils::DateTime	TimeAcquired;
+};
+
+// -----------------------------------------------------------------------------
+
+struct ProgressionData
+	: public Utils::Serialisation::ISerialisable
+{
+public:
+
+	ProgressionData();
```

Comment:
Same here - can be = default

Reply:
Hang on, since there's no other `ProgressionData` constructors defined, I can just remove it and have it use an implicitly defined default constructor, right?